doc/explanation/microcloud.rst

```diff
@@ -26,8 +26,30 @@ Networking
 
 By default, MicroCloud uses MicroOVN for networking, which is a minimal wrapper around OVN (Open Virtual Network).
 
-MicroOVN requires an uplink network that is an actual L2 subnet (which is usually not the case in a virtual cloud environment).
-In addition, MicroOVN requires its own dedicated network interface, for example, a dedicated physical network interface, a VLAN, or a virtual function on an :abbr:`SR-IOV (Single root I/O virtualisation)`-capable network interface.
+For outside connectivity, MicroOVN requires an uplink network.
+This uplink network must be an actual L2 subnet (which is usually not the case in a virtual cloud environment).
```

Comment:
"L2 subnet" is a contradiction in networking terms.

This is because "subnet" is an IP (internet protocol) concept, and IP is layer 3 (network layer).
L2 (or layer 2) indicates the layer below the network layer, which is the data link layer, most commonly Ethernet.

This link is useful to explain the different concepts:
https://www.cloudflare.com/en-gb/learning/network-layer/what-is-the-network-layer/

Although any mention of layers can become confusing, as that link describes, there are different definitions of layers.

I wonder if we shouldn't avoid mentioning layers at all, and instead be explicit as to what we require from the uplink network.

Namely we expect the uplink network interface to support broadcast and multicast traffic so that IP adverts from the OVN virtual router(s) are understood by the uplink network and can then ensure packets for the router make their way to the active chassis member.

Reply:
The reason we've said about layer 2 networking in the past is because a proper Ethernet network generally allows for ARP and NDP packets to be broadcast to all nodes on the network, including the uplink router, and this allows for the OVN virtual router(s) external IP on the uplink network to be migrated between LXD cluster members in the case that one LXD cluster member goes down and the others need to take over the role of providing external connectivity to the OVN network(s) by taking control of the virtual IP(s).